src/main/java/org/generationcp/middleware/api/germplasm/update/GermplasmUpdateService.java

```diff
@@ -0,0 +1,11 @@
+package org.generationcp.middleware.api.germplasm.update;
+
+import org.generationcp.middleware.domain.germplasm.GermplasmUpdateDTO;
+
+import java.util.List;
+
+public interface GermplasmUpdateService {
```

Comment:
I agree, we can have one service for import and update. Will adjust my code after your branch is merged.

Reply:
This is done. I deleted the GermplasmUpdateService and moved the logic to GermplasmService. 

I didn't reuse the GermplasmImportRequestDto.GermplasmDto for importing germplasm updates since it requires gid and creation date must be in yyyyMMdd format.